recipeapp/src/Recipes/SearchBar/SearchBar.js

```diff
@@ -0,0 +1,26 @@
+import React,{Component} from 'react';
```

Comment:
I expected you to have an `index.js` file within the `SearchBarDirectory`. The tree should look something like this

```
SearchBar
    | - SearchBar.js
    | - SearchBar.css
    | - index.js
```

The `index.js` file would have the sole purpose of exporting the `SearchBar`component to the outside world. `index.js` could look something like this

```JavaScript
export { SearchBar } from './SearchBar';
```

Reply:
I have made the changes